examples/aes/src/aes.rs

```diff
@@ -8,6 +8,8 @@ pub const ROUNDS: usize = 10;
 pub const KEY_SCHEDULE_LENGTH: usize = 176;
 pub const ITERATIONS: usize = 40;
 
+pub const INVALID_KEY_EXPANSION_INDEX: u8 = 1u8;
```

Comment:
You can replace this `u8` error code by a proper enum describing all the ways AES can fail if you want.

Reply:
Right, I somehow started with `u8`s and didn't switch 😬 enums are of course nicer.